### ISSUE TITLE
Unable to activate Safari extensions

### ISSUE THREAD

**Issue by jannis-baum** [godbout/Wooshy.docs#87]
This is weird and I can't reproduce it right now:

When I was trying to activate any Safari extension, I got a popup saying "Safari detected an app or service that interfered with clicking." and that I can't turn on the extension because of it. It linked to [this deprecated support document from Apple](https://support.apple.com/en-us/HT211233).

Once I quit Wooshy it worked again. Then I reopened Wooshy and now I no longer have the issue (maybe reboot will bring it back up? don't have time to test right now). Feel free to close if you don't think it's worth investigating!

**Comment by godbout** [comment]
hmm. i have the message yes, but it just asks for your password/Touch ID:

https://user-images.githubusercontent.com/121373/203743560-1984b589-e7d1-4eca-9932-1f6cc94e675d.mp4

what version of macOS and Safari are you on?

**Comment by jannis-baum** [comment]
Interesting! I tried clicking it with the mouse (shame on me) but it didn't give me the option to put in a password.

I'm on macOS 13.0.1 & Safari 16.1 with my Macbook's lid closed (i.e. Touch ID disabled). But can't reproduce the issue right now.

**Comment by jannis-baum** [comment]
Yes, I just figured out the same thing as well... I guess this is more of an issue for Apple then?😂 Feel free to close or decide what to do with it

**Comment by godbout** [comment]
i'll report to Apple but there may be something to be done on Ws's side. will check.
